### PR TITLE
battery voltage sensor: make ADC attenuation configurable

### PR DESCRIPTION
**Pull request by thiasB**
At least on my system (LoPy4 + Expansion Board 3.0) the hard coded 6db attenuation wasn't enough to not feed the ADC below it's maximum allowed voltage. By using 11db it works fine. Change preserves the attenuation default of 6db.

### CODE REVIEW

File: terkin/sensor/system.py [hunk 1]
```diff
@@ -135,13 +135,25 @@ def setup(self):
         self.pin = self.settings.get('pin')
         self.resistor_r1 = self.settings.get('resistor_r1')
         self.resistor_r2 = self.settings.get('resistor_r2')
+        self.adc_attenuation_db = self.settings.get('adc_attenuation_db')
```
Comment (amotl) [inline terkin/sensor/system.py]: Can we say `self.settings.get('adc_attenuation_db', 6.0)` here in order to make this backward-compatible?

**Comment by amotl** [comment]
Thanks Matthias!

Besides addressing the comment above, it would be great to put the configuration blueprint also inside `settings.example-bob.py`.

**Comment by thiasB** [comment]
Happy?

**Comment by amotl** [comment]
Thanks a bunch!

**Pull request merged**